components/Table/playerMatchesColumns.js

```diff
@@ -1,49 +1,60 @@
 import React from 'react';
 
+const defaultSort = (array, field) => array.sort((a, b) => {
+  if (a[field] < b[field]) return -1;
+  if (a[field] > b[field]) return 1;
+  return 0;
+});
+
+const useOriginalValueSort = (array, field) => {
+  const origField = field.slice(0, field.lastIndexOf('Display'));
+  return defaultSort(array, origField);
+};
+
 export default [{
```

Comment:
I think we should pass field as the name of the field (no "Display"), and pass the display function as one of the props (dispFn)